.pre-commit-config.yaml

```diff
@@ -1,56 +1,75 @@
 default_language_version:
```

Comment:
This formatting change is by the `prettier`, right?

Reply:
Yes